index.html

```diff
@@ -65,7 +65,131 @@ <h2 class="main-title main-section-subtitle">I'm a software<br> developer</h2>
+        <img class="frame2" src="assets/images/Frame2.png" alt="frame 2">
+        <div class="aboutme-text-area ">
+                <h1  class="aboutme-title">About me</h1>
+            <p class="aboutme-paragraph">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                &#39;t hestiate to contact me.
+            </p>            
+            <a><div class="btn-get-resume">Get My Resume</div></a>
+        </div>
+        <div class="aboutme-cards-area">
+            <div class="aboutme-card-item">
+                <div class="aboutme-card-logo">
+                    <img class="aboutme-logo" src="assets/icons/aboutme-icon1.png" alt="aboutme logo 1">
+                </div>
+                <div class="aboutme-card-details">
+                    <h2 class="aboutme-card-title">Languages</h2>
+                    <div class="aboutme-tags-container">
+                        <div class="aboutme-tags-item">JavaScipt</div>
+                        <div class="aboutme-tags-item">Ruby</div>
+                        <div class="aboutme-tags-item">Html</div>
+                        <div class="aboutme-tags-item">css</div>
+                    </div>
+                </div>
+            </div>
+            <div class="aboutme-card-item">
+                <div class="aboutme-card-logo">
+                    <img class="aboutme-logo" src="assets/icons/aboutme-icon2.png" alt="about me logo 2">
+                </div>
+                <div class="aboutme-card-details">
+                    <h2 class="aboutme-card-title">Frameworks</h2>
+                    <div class="aboutme-tags-container">
+                        <div class="aboutme-tags-item">Bootstrap</div>
+                        <div class="aboutme-tags-item">Ruby on rails</div>
+                        <div class="aboutme-tags-item">RSpec</div>
+                        <div class="aboutme-tags-item">Capybara</div>
+                        <div class="aboutme-tags-item">Selenium</div>
+                    </div>
+                </div>
+            </div>
+            <div class="aboutme-card-item">
+                <div class="aboutme-card-logo">
+                    <img class="aboutme-logo" src="assets/icons/aboutme-icon3.png" alt="about me logo 3">
+                </div>
+                <div class="aboutme-card-details">
+                    <h2 class="aboutme-card-title">Skills</h2>
+                    <div class="aboutme-tags-container">
+                        <div class="aboutme-tags-item">Codekit</div>
+                        <div class="aboutme-tags-item">GitHub</div>
+                        <div class="aboutme-tags-item">Codepen</div>
+                        <div class="aboutme-tags-item">Gitlab</div>
+                        <div class="aboutme-tags-item">Terminal</div>
+                    </div>
```

Comment:
- [ ] Same thing as before here, please make sure to convert these divs into list elements using `ul`'s and `li`'s.